eth2util/signing/signing.go

```diff
@@ -29,10 +29,11 @@ import (
 type DomainName string
 
 const (
-	DomainBeaconProposer DomainName = "DOMAIN_BEACON_PROPOSER"
-	DomainBeaconAttester DomainName = "DOMAIN_BEACON_ATTESTER"
-	DomainRandao         DomainName = "DOMAIN_RANDAO"
-	DomainExit           DomainName = "DOMAIN_VOLUNTARY_EXIT"
+	DomainBeaconProposer        DomainName = "DOMAIN_BEACON_PROPOSER"
+	DomainBeaconBuilderProposer DomainName = "DOMAIN_BEACON_BUILDER_PROPOSER"
```

Comment:
mmm not sure about this. These domains are defined in the eth2 spec (it isn't our own thing). I guess blinded blocks are signed with normal  `DomainBeaconProposer` actually. Can you confirm? If so, you can then remove this.

Reply:
yep you're correct https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md double checked on the eth research discord and they confirmed "DOMAIN_BEACON_PROPOSER" is to be used on both